spec/index.html

```diff
@@ -1825,7 +1823,7 @@ <h2>Changes between RDF 1.1 and RDF 1.2</h2>
     <li>Added <a href="#section-dataset-quad" class="sectionRef"></a>
       for informative definition of a <a>quad</a>.</li>
     <li>Added <a href="#section-quoted-triples" class="sectionRef"></a>
-      <s>and definitions for <a>quoted triple</a> and <a>asserted triple</a></s>.
+      <s>and definitions for <a>quoted triple</a></s> and definition for <a>asserted triple</a></s>.
```

Comment:
IIUC, the term "quoted triple" is now obsolete. It can simply be removed when it's complete, but the whole section is affected. It probably becomes "triple term", but that's still a bit up in the air.

Reply:
There's now an orphan `</s>`. I think this suggestion makes the right correction, but I'm not certain.
```suggestion
      <s>and definitions for <a>quoted triple</a></s>
      and definition for <a>asserted triple</a>.
```

Reply:
That strike-through text will eventually go away when we remove quoted triple. But, this is an appropriate change for now.